src/Compiler/AbstractIL/ilwrite.fs

```diff
@@ -3756,7 +3756,7 @@ let writePdb (
                         let pdbfileInfo = FileInfo(pdbfile).FullName
 
                         // If pdbfilepath matches output filepath then error
-                        if String.Compare(outfileInfo, pdbfileInfo, StringComparison.InvariantCulture) = 0 then
+                        if outfileInfo = pdbfileInfo then
```

Comment:
What's the particular motivation for this one?

Reply:
File systems usually doesn't normalize Unicode in paths, InvariantCulture does

Reply:
Those paths are CLI arguments, the logic to compare them should stay case insensitive like it was, to keep the behavior the same.

Reply:
@T-Gro it was `InvariantCulture`, not `InvariantCultureIgnoreCase`

Reply:
@T-Gro do you suggest to actually make it case insensitive?